### ISSUE TITLE
Is it spread? No... It's weird sorting - I'd like to see bid / asks closer to each other please.

### ISSUE THREAD

**Issue by stefek99** [etherdelta/etherdelta.github.io#57]
![image](https://user-images.githubusercontent.com/249703/29347849-72d4e5e8-8247-11e7-892a-fbe8a67ad54c.png)

**Comment by 1337cookie** [comment]
No, that's the spread, its inverted (negative). There's no match making due to the nature of etherdelta being a smart contract. So if you put up bids and asks they don't execute against each other and it can take a little while for orders to disappear from the order book.

**Comment by Texxer** [comment]
take a little while for orders to disappear... little is good. try to find a trade that is not already taken... good luck.

"You cannot trade this order because it already traded. Someone else already traded this order and the order book hasn't updated yet."

"You tried to send an Ethereum transaction but there was an error."

Completely unusable.

I like the sorting though...
